src/components/forms/TextField.tsx

```diff
@@ -208,7 +208,7 @@ export function createInput(Component: typeof TextInput) {
               paddingBottom: 2,
             }),
             {
-              lineHeight: a.text_md.lineHeight * 1.1875,
+              lineHeight: a.text_md.fontSize * 1.1875,
```

Comment:
Removed `lineHeight` from atoms, so needed to use `fontSize` (same value) for this hack in inputs